.circleci/config.yml

```
@@ -0,0 +1,126 @@
+#
+# Copyright (c) 2021, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+version: 2.1
+
+executors:
+  daml-docker-executor:
+    docker:
+      - image: cimg/openjdk:11.0
+
+commands:
+  install_sdk:
+    description: "Install Daml SDK"
+    steps:
+      - run:
+          name: Install Daml SDK
+          command: |
+            curl -sSL https://get.daml.com/ | sh /dev/stdin ${SDK_VERSION}
+            # idea from https://circleci.com/docs/2.0/env-vars/
+            >> $BASH_ENV echo 'export PATH="$HOME/.daml/bin:$PATH"'
+  install_yq:
+    description: "Install yq from binary"
+    steps:
+      - run:
+          name: Install yq
+          command: |
+            mkdir -p $HOME/yq
+            curl -L https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 -o $HOME/yq/yq &&\
+              chmod +x $HOME/yq/yq &&\
+              echo 'export PATH=$HOME/yq/yq:$PATH' >> $BASH_ENV
+  install_gh:
+    description: "Install GitHub Cli"
+    steps:
+      - run:
+          name: Install gh
+          command: |
+            curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
+            echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+            sudo apt update
+            sudo apt install gh
+  set_sdk_version:
+    description: "Set Daml SDK version environment version"
+    steps:
+      - run:
+          name: Set Daml SDK version
+          command: echo 'export SDK_VERSION=$(yq e '.sdk-version' daml.yaml)' >> $BASH_ENV
+  run_release:
+    description: "Run Release"
+    steps:
+      - run:
+          name: "Get target release version"
+          command: echo 'export VERSION=$(yq e '.version' daml.yaml)' >> $BASH_ENV
+      - run:
+          name: "Validate target release version"
+          command: |
+            if [ `git tag | grep ${VERSION} | wc -l` -eq 0 ]; then
```

Perhaps this is safer:

```suggestion
            if [ `git tag` == $VERSION ] ; then
```

it would break if you have e.g. `v2.0.0-rc1`?

---

I hadn't considered that use case. I'll update the script